apps/nextra/pages/en/build/smart-contracts/_meta.tsx

```diff
@@ -23,8 +23,8 @@ export default {
     type: "separator",
     title: "Aptos Standards",
   },
-  objects: {
-    title: "Object",
+  "aptos-objects": {
+    title: "Objects",
```

Comment:
I tried keeping this route as objects, but if you go to https://aptos.dev/en/build/smart-contracts/objects/creating-objects and refresh, it switches the page to `using-objects` and I'm not sure why. Changing the name of the route to something other than `objects` fixes it though.

Reply:
Maybe there's a permalink in the redirects.  A lot of them are set as permanent.